stable/openvpn/Chart.yaml

```diff
@@ -3,7 +3,7 @@ description: A Helm chart to install an openvpn server inside a kubernetes clust
   generation is also part of the deployment, and this chart will generate client keys
   as needed.
 name: openvpn
-version: 3.12.5
+version: 3.12.6
```

Comment:
This is a big enough change to make it a minor version bump IMO.

Reply:
That's cool :) so does that mean this becomes 3.13.0? 3.13.1?

Reply:
Set it to 3.13.0

Reply:
Yep that’s right